pkg/cmd/cli/describe/printer.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED"}
+	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED", "REASON"}
```

Comment:
To do that and still show columns aligned with headers we first need to solve https://github.com/openshift/origin/issues/4950